pycbc/coordinates.py

```diff
@@ -58,8 +58,12 @@ def cartesian_to_spherical_azimuthal(x, y):
     """
     if type(y) is int:
         y = float(y)
-    return numpy.arctan(y / x)
-
+    phi = numpy.arctan2(y, x)
+    if phi < 0:
+        phi = 2 * numpy.pi - abs(phi)
+    elif phi > 2 * numpy.pi:
+        phi = phi - 2 * numpy.pi
+    return phi
```

Comment:
This will break being able to use this function on numpy arrays, which every other coordinate transformation and conversion can do. Do this instead:
```
return phi %(2*numpy.pi)
```

Reply:
Changed. Thanks.